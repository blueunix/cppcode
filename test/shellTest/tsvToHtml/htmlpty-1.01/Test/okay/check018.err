:14:HTML grammars do not permit an end tag for the tag name [AREA]; please consider deleting it
:16:HTML grammars do not permit an end tag for the tag name [ATOP]; please consider deleting it
:18:HTML grammars do not permit an end tag for the tag name [BASE]; please consider deleting it
:20:HTML grammars do not permit an end tag for the tag name [BASEFONT]; please consider deleting it
:22:HTML grammars do not permit an end tag for the tag name [BR]; please consider deleting it
:24:HTML grammars do not permit an end tag for the tag name [CHOOSE]; please consider deleting it
:26:HTML grammars do not permit an end tag for the tag name [COL]; please consider deleting it
:28:HTML grammars do not permit an end tag for the tag name [FRAME]; please consider deleting it
:30:HTML grammars do not permit an end tag for the tag name [HR]; please consider deleting it
:32:HTML grammars do not permit an end tag for the tag name [IMG]; please consider deleting it
:34:HTML grammars do not permit an end tag for the tag name [INPUT]; please consider deleting it
:36:HTML grammars do not permit an end tag for the tag name [ISINDEX]; please consider deleting it
:38:HTML grammars do not permit an end tag for the tag name [LEFT]; please consider deleting it
:42:HTML grammars do not permit an end tag for the tag name [META]; please consider deleting it
:44:HTML grammars do not permit an end tag for the tag name [NEXTID]; please consider deleting it
:46:HTML grammars do not permit an end tag for the tag name [OVER]; please consider deleting it
:48:HTML grammars do not permit an end tag for the tag name [OVERLAY]; please consider deleting it
:50:HTML grammars do not permit an end tag for the tag name [PARAM]; please consider deleting it
:52:HTML grammars do not permit an end tag for the tag name [RIGHT]; please consider deleting it
:54:HTML grammars do not permit an end tag for the tag name [TAB]; please consider deleting it
:56:HTML grammars do not permit an end tag for the tag name [WBR]; please consider deleting it
:63:obsolete feature: nested anchor tags <A ...> ... <A ...> ... </A> ... </A>
:69:</BODY> at level 0 should be at level 1
