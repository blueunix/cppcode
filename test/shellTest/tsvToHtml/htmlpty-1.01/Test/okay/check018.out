<!-- -*-html-*- -->

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 4.0 Transitional//EN" >

<HTML>
    <HEAD>
        <TITLE>
        </TITLE>
        <LINK REV="made" HREF="mailto:beebe@plot79.math.utah.edu">
    </HEAD>
    <BODY>
        <AREA>
        this is forbidden AREA content
    </AREA>
<ATOP> this is forbidden ATOP content </ATOP>
<BASE>
this is forbidden BASE content
</BASE>
<BASEFONT>
this is forbidden BASEFONT content
</BASEFONT>
<BR>
this is forbidden BR content</BR>
<CHOOSE> this is forbidden CHOOSE content </CHOOSE>
<COL>
this is forbidden COL content
</COL>
<FRAME>
this is forbidden FRAME content
</FRAME>
<HR>
this is forbidden HR content
</HR>
<IMG>
this is forbidden IMG content
</IMG>
<INPUT>
this is forbidden INPUT content
</INPUT>
<ISINDEX>
this is forbidden ISINDEX content
</ISINDEX>
<LEFT> this is forbidden LEFT content </LEFT>
<LINK>
this is forbidden LINK content
</LINK>
<META>
this is forbidden META content
</META>
<NEXTID>
this is forbidden NEXTID content
</NEXTID>
<OVER> this is forbidden OVER content </OVER>
<OVERLAY>
this is forbidden OVERLAY content
</OVERLAY>
<PARAM>
this is forbidden PARAM content
</PARAM>
<RIGHT> this is forbidden RIGHT content </RIGHT>
<TAB> this is forbidden TAB content </TAB>
<WBR>
this is forbidden WBR content
</WBR>
<P>
    Nested anchors became illegal at grammar level 3.2, so we should see
    a warning for this:
    <A HREF="outer">
        outer
        <A HREF="inner">
            inner
        </A>
        and outer again
    </A>
</P>
</BODY>
</HTML>
